### DISCUSSION TITLE
Github contributions

### DISCUSSION THREAD

**Discussion by komus-Israel** [community/community#9210] [Other Feature Feedback, Questions, & Ideas]
I added an email to my github profile and my total contribution decreased drastically from over 800 to 536. What's wrong with github contributions. I've removed the email I added to my profile but my number of contributions has not been updated

**Comment by Arhell** [comment]
Hello.
If you used another email, then you also need to add it, if you deleted it, then everything that you did will not be shown in your profile.

and more info in docs
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-graphs-on-your-profile/viewing-contributions-on-your-profile
